index.html

```diff
@@ -42,5 +42,119 @@
 
     </header>
 
-  </body>
+    <main class="main-index">
+      <h1 class="visually-hidden">Интернет-магазин device</h1>
```

Comment:
```suggestion
      <h1 class="visually-hidden">Интернет-магазин Device</h1>
```
Все тексты в разметке должны быть написаны по правилам языка страницы